src/constants.ts

```diff
@@ -5,7 +5,6 @@ const packageDotJson = require("../package.json");
 export const PROGRAM_NAME = packageDotJson.name;
 export const PROGRAM_VERSION = packageDotJson.version;
 export const USER_AGENT = `CrossSeed/${PROGRAM_VERSION}`;
-export const TORRENT_TAG = "cross-seed";
```

Comment:
I think leaving this intact and falling back to TORRENT_TAG in commander would be a better approach, as it keeps our constants in the same place, rather than having the default inline.